campaigns/mofo-eoy-2015/snippet.html

```diff
@@ -250,15 +250,20 @@
           snippet.addEventListener('show_snippet', function() {
               (function () {
                 document.querySelector("body").classList.add("background-{{ snippet_id }}");
-                var donationURL = '{{ donationFormURL }}';
+                var donationURL = '{{ donationFormURL|safe }}';
                 var currencyCode = '{{ currencyCode|default('usd', true) }}';
-                var extraParams = 'snippet_name={{ snippet_id }}';
```

Comment:
Apologies you'll have to keep this and ignore my previous comment. The service does indeed set `snippet_name` but the way you manipulate the params when the buttons are clicked it gets removed again.